openapi-python-templates/api.mustache

```diff
@@ -115,8 +115,8 @@ class _{{classname}}:
             {{#queryParams.0}}params=query_params,{{/queryParams.0}}
             {{#headerParams.0}}headers=headers,{{/headerParams.0}}
             {{#cookieParams.0}}cookies=cookies,{{/cookieParams.0}}
-            {{#formParams.0}}data=data,{{/formParams.0}}
-            {{#fileParams.0}}files=files,{{/fileParams.0}}
+            {{#formParams.0}}data=data,
+            files=files{{/formParams.0}}
```

Comment:
```suggestion
            files=files{{/formParams.0}} or None
```
That way we don't pass an empty dict, which seems to affect the way the request is serialized. (I'm not 100% sure about this though.)

Does that seem okay to you?

Reply:
Yes it does, None will send the form as application/x-www-form-urlencoded, {} will send it as multipart/form.

Reply:
Aha, thanks for checking that! This is great.